packages/accounts-oauth/oauth_common.js

```
@@ -21,6 +21,15 @@ Accounts.oauth.registerService = name => {
   }
 };
 
+// Recreate OAuth service indexes when the users collection changes.
+if (Meteor.server) {
+  Accounts.onUsersCollectionChanged(users => {
+    for (const name of Object.keys(services)) {
+      users.createIndexAsync(`services.${name}.id`, {unique: true, sparse: true});
```

ESLint

[error] Replace `unique:·true,·sparse:·true` with `⏎········unique:·true,⏎········sparse:·true,⏎······`

(prettier/prettier)